Feat/script gen pdf improvement (#109)
privacy-scaling-explorations/anon-aadhaar: README.md

```bash
yarn pdf:pcd
```

> You can generate more pdf files have a same certificate by command: 

Reviewer: You can generate several PDF files with the same certificate by running the following command:
```bash
yarn pdf:pcd other-file.pdf #other-file can be any name you want to give to the file
```

> You can generate more pdf files have a same certificate by command: 

Author: ```suggestion
You can generate several PDF files with the same certificate by running the following command:
```

```bash

> yarn pdf:pcd other-file.pdf #other-file can be any name you want

Author: ```suggestion
yarn pdf:pcd other-file.pdf #other-file can be any name you want to give to the file
```
